src/scenes/game/GuessForm.tsx

```diff
@@ -11,14 +11,50 @@ export const GuessForm = ({
   onGuessUpdated,
```

Comment:
should we change those to present tense ```onGuessUpdate``` etc?

Reply:
good catch; I'll make that change and include it in this PR